### PR TITLE
Fix mini bug

### PR DESCRIPTION
**Pull request by LezaiYa**

### CODE REVIEW

File: Patches/PlayerControlPatch.cs [hunk 1]
```diff
@@ -2770,7 +2770,7 @@ public static void Postfix(PlayerControl __instance)
                                         Logger.Info($"年龄增加1", "Child");
                                         if (Mini.UpDateAge.GetBool())
                                         {
-                                            player.Notify(GetString("MiniUp"));
+                                            if (player.Is(CustomRoles.NiceMini)) player.Notify(GetString("MiniUp"));
```
Comment (LezaiYa) [inline Patches/PlayerControlPatch.cs:2773]: @Tommy-XL Why remove _if (player.Is(CustomRoles.Nice/Evil Mini))_    ?

Comment (Tommy-XL) [reply Patches/PlayerControlPatch.cs:2773]: Because this code is already in `case CustomRoles.NiceMini:` so I didn't see the point in additional checking

Comment (LezaiYa) [reply Patches/PlayerControlPatch.cs:2773]: Without the extra check, it will cause the player to not see the mini's age

Comment (LezaiYa) [reply Patches/PlayerControlPatch.cs:2773]: It's a bug that shouldn't be there ._.

**Review by 0xDrMoe — APPROVED** [review]

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**